prep-javascript-operators-and-expressions/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Prep Javascript Operators and Expressions</title>
+    <script src="main.js"></script>
```

Comment:
We always want to put our `script` tags at the bottom of the `<body>`. We want the browser to first load all of the HTML, and then the JavaScript